crypto/fipsmodule/self_check/self_check.c

```diff
@@ -1198,10 +1198,15 @@ int BORINGSSL_self_test(void) {
       // When requested to run self tests, also run the lazy tests.
       !boringssl_self_test_rsa() ||
       !boringssl_self_test_ecc() ||
-      !boringssl_self_test_ffdh()) {
+      !boringssl_self_test_ffdh() ||
+      !boringssl_self_test_sha256() ||
+      !boringssl_self_test_hmac_sha256()) {
```

Comment:
They're not lazy, they're previously only called in the constructor. To simplify my testing, and in the spirit of BORINGSSL_self_test checking all the operational algorithms I added them. I like your spot better, I didn't see that originally.

Reply:
I agree, because I got confused and thought they sha256 and hmac_sha256 were lazy.
I'd say "run lazy tests and SHA-256 and HMAC-SHA256 which are part of the integrity test."